code/modules/mob/living/carbon/human/species.dm

```diff
@@ -2135,6 +2137,11 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 /datum/species/proc/GiveSpeciesFlight(mob/living/carbon/human/H)
 	if(flying_species) //species that already have flying traits should not work with this proc
 		return
+	if(ismoth(H) && H.dna.features["moth_wings"]) //if we have mothwings, we keep them and not grow new ones
```

Comment:
why are you checking if we're a moth? this is givespeciesflight. a species proc. just override it on the moth species.

Reply:
it won't work without it, but i should probably add a check for a mothwing organ instead